modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -14155,6 +14159,7 @@ star-off=Star Off
 stars=Stars
 start=Start
 start-a-new-workflow=Start a New Workflow
+start-creating-a-sort-to-display-specific-data=Start creating a Sort to display specific Data.
```

Comment:
Maybe Brian complains about the words `Sort` and `Data` because they are capitalized. Could you see it with your manger?